design-docs/Expiration.md

```diff
@@ -125,3 +125,90 @@ return resolvedField
 ```
 
 Note: the `maxStale` duration is to allow for a per-operation override of the max age / expiration date. 
+
+## Declarative API
+
+### Schema directives
+
+These directives will land in a `cache` v0.1 [Apollo Spec](https://specs.apollo.dev/).
```

Comment:
How much could we reuse the server side`@cacheControl`?

https://www.apollographql.com/docs/apollo-server/performance/caching/

Reply:
> `scope` is irrelevant on the client (can simply be ignored)

We _could_ have cross-user cache. For an example, in Confetti, all the session data is `PUBLIC`, but the bookmarks are `PRIVATE`. We're not there yet but could be something?

Agree ignoring it for now is fine.

> we still need a "Field" version of the directive

Yea, either `@cacheControlField()` or field definition extensions in the GraphQL 2.0 🤓  

Overall, agree that we could model things more precisely but the prospect of getting all of these for free from the backend is pretty enticing